backend/typescript/migrations/create_users_table.ts

```diff
@@ -0,0 +1,42 @@
+import { DataType } from "sequelize-typescript";
+
+import { Migration } from "../umzug";
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().createTable("users", {
+    id: {
+      type: DataType.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    auth_id: {
+      type: DataType.STRING,
+      allowNull: false,
+    },
+    role_type: {
+      type: DataType.ENUM("Admin", "Subscriber", "Author"),
+      allowNull: false,
+    },
+    first_name: {
+      type: DataType.STRING,
+      allowNull: false,
+    },
+    last_name: {
+      type: DataType.STRING,
+      allowNull: false,
+    },
+    email: {
+      type: DataType.STRING,
+      allowNull: false,
+    },
+    active: {
+      type: DataType.BOOLEAN,
+      allowNull: false,
+    },
+    createdAt: DataType.DATE,
```

Comment:
Please also add an updatedAt field
```suggestion
    createdAt: {
      allowNull: false,
      type: DataType.DATE,
    },
    updatedAt: {
      allowNull: false,
      type: DataType.DATE,
    },
```